src/ossfs/async_oss.py

```diff
@@ -156,7 +156,10 @@ async def _call_oss(
             if not method:
                 method = getattr(aiooss2, method_name)
                 logger.debug("CALL: %s - %s - %s", method.__name__, args, kwargs)
-                out = method(service, *args, **kwargs)
+                if method_name =="resumable_upload":
+                    out = await method(service, *args, **kwargs)
+                else:
+                    out = method(service, *args, **kwargs)
```

Comment:
Are there any sync methods here?

Reply:
Yes, sometimes it return `AioObjectIterator` which can not be awaited

Reply:
for example, https://github.com/CNLHC/ossfs/blob/ce49c91976fd30c8b5f223308d5618d6e414d99b/src/ossfs/async_oss.py#L211

and

https://github.com/CNLHC/ossfs/blob/ce49c91976fd30c8b5f223308d5618d6e414d99b/src/ossfs/async_oss.py#L316